LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2020 Denis Olehov
```

Comment:
@Vinzent03 we must add your name here as well, and it should be before mine :D

Reply:
Given that your handle is technically PII, it is possible to use it (according to my quick google search :D)